libiocage/lib/Jail.py

```diff
@@ -852,9 +890,26 @@ def __dir__(self):
         properties = set()
 
         for prop in dict.__dir__(self):
-            if prop.startswith("_get_"):
-                properties.add(prop[5:])
-            elif not prop.startswith("_"):
+            if not prop.startswith("_"):
                 properties.add(prop)
 
         return list(properties)
+
+
+class Jail(JailGenerator):
+
+    _class_host = libiocage.lib.Host.HostGenerator
+
+    def start(self, *args, **kwargs):
+        return list(JailGenerator.start(self, *args, **kwargs))
+        # libiocage.lib.helpers.print_event_generator(
```

Comment:
Since this code doesn't exist in master anyways, we should just remove the commented code. Unless you didn't accidentally push it?

Reply:
I was following different approaches to wrap the generators, because I think it's important that libiocage can work in both modes. Of course I'm trying to keep the overhead caused by wrapping as low as possible. Expect this code to be removed before cleaning up for a merge!